.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           bin/whippet-application-tagger.sh
 
       - name: Keep workflow alive
-        uses: gautamkrishnar/keepalive-workflow@c85efc9567a3dadb9b9d4e116aa891f76c0ef7e6
+        uses: gautamkrishnar/keepalive-workflow@1.1.0
```

Comment:
I think we'd deliberately pinned to a commit hash rather than a version tag here because it's an external workflow that will be able to make commits to the repo, and the intention was to avoid the possibility of a malicious actor getting control of the workflow and moving the tag to a different commit that could do something bad. 

Although thinking about it some more, I'm not convinced it's any more of a risk in this case than when we use any other external workflow.